.gitlab-ci.yml

```diff
@@ -66,6 +66,7 @@ variables:
   S3_OMNIBUS_CACHE_BUCKET: dd-ci-datadog-agent-omnibus-cache-build-stable
   USE_S3_CACHING: --omnibus-s3-cache
   S3_DSD6_URI: s3://dsd6-staging/linux
```

Comment:
`S3_DSD6_URI` should not have `linux` in the path anymore. We should rename it, or just do the concatenation in the jobs.

Note that doing env vars concatenation will likely not work. E.g.: 
```yaml
S3_DSD6_BASE: s3://dsd6-staging
S3_DSD6_LINUX_URI: $S3_DSD6_BASE/linux
S3_DSD6_WINDOWS_URI: $S3_DSD6_BASE/windows
```
^^ I believe won't fly.

Reply:
Just updated this 😄